CHANGELOG.md

```diff
@@ -159,6 +159,7 @@ Please refer to the [Breaking changes](https://developers.arcgis.com/javascript/
 * [Esri Community - 1272209](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/cimsymbol-not-tiling-correctly/m-p/1272209): Fixed an issue where some CIM symbols with a CIMHatchFill symbol layer were not aligned at tile boundaries at large scales.
 * [Esri Community - 1275187](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/cimsymbol-not-working-in-sceneview-3d/m-p/1275187): Fixed an issue where CIM symbols using icon fonts were not rendering properly in 3D.
 * Fixed an accessibility issue where the Bookmarks widget's drag/reorder tooltip was not displayed when in focus for keyboard-only users.
+* Fixed an issue where the ImageMediaInfo's refreshInterval property was not sending requests to refresh the image.
```

Comment:
I wasn't quite sure what this fix meant at first - could you specify somewhere that this is for Popup images?